saml-idp.py

```diff
@@ -149,11 +149,35 @@ def make_assertion(resp, acs, irt, iss, username, validity):
                attrib={'InResponseTo': irt,
                        'NotOnOrAfter': tsnow(validity),
                        'Recipient': acs})
+    # new stuff
```

Comment:
Not a blocker, maybe call this "Okta new stuff" just for later support! :)

Reply:
good call